grunt/tasks/javascript.js

```diff
@@ -0,0 +1,57 @@
+module.exports = function(grunt) {
+
+	var convertSlashes = /\\/g;
+	
+    grunt.registerMultiTask('javascript', 'Compile JavaScript files', function() {
+
+  		var requirejs = require('requirejs');
+		var _ = require('underscore');
+		var path = require("path");
+		var fs = require("fs");
+		var done = this.async();
+		var options = this.options({});
+
+		if (options.plugins) {
+			var pluginsClientSidePatch = 'requirejs.config({map: { "*": { "extensions/extensions":"'+options.pluginsModule+'","menu/menu":"'+options.pluginsModule+'","theme/theme":"'+options.pluginsModule+'","components/components":"'+options.pluginsModule+'" } } });';
+
+			if (!fs.existsSync(options.pluginsPath)) {
```

Comment:
I think `existsSync()` has been deprecated.  Can you consider using `statSync()` here instead?